content/docs/user-guide/project-structure/dvcyaml-files.md

```diff
@@ -47,23 +47,8 @@ are defined at the file level and include all parameters in the file. See
 The list of `plots` contains one or more user-defined `dvc plots`
 configurations. Every plot must have a unique ID, which may be either a file or
 directory path (relative to the location of `dvc.yaml`) or an arbitrary string.
```

Comment:
Just a thought: wouldn't it make everyone's life easier if only paths were accepted as plot IDs?

Reply:
Then how would you define a plot that uses multiple paths? Or define multiple plots that use the same path?